Sources/PayPalWebPayments/PayPalVaultRequest.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// A request to vault a PayPal payment method
+public struct PayPalVaultRequest {
+
+    /// PayPal URL for the approval web page
+    public let url: URL
```

Comment:
Slight tangent here - but do you have context on why we have to require the URL? I assume some limitation with the API. Do we have it documented somewhere?

Reply:
PayPal vault works very differently than Card vault in that we just simply launch the url for API to be called outside of our SDK and we get results back. There is documentation for v3/vault/setup-tokens with PAYPAL WALLET in PPaas. Not sure if that is what you are asking.

Reply:
Do we document things that happens on merchant's server side or do we leave that to our developer docs?
I like your suggestion of describing it in our SDK in this case because it's not self-explanatory where it comes from.

Reply:
I think it could help a merchant who maybe doesn't know where this URL comes from or which URL to pass in. I could certainly see a merchant passing in the wrong URL here so think giving some guidance would be beneficial.

Reply:
addressed here: dadbd5de17e6cc2524f7128ecedc7968d3dff3ef